graphql/schema.graphql

```diff
@@ -643,9 +657,11 @@ type GroupedProjects {
 
 type Project {
   identifier: String!
+  id: String!
   displayName: String!
   repo: String!
   owner: String!
+  patches(patchesInput: PatchesInput!): Patches!
```

Comment:
I really like this pattern of adding queryable fields to our types. I think doing this more often will allow us to  really take advantage of graphql's capabilities, instead of using it as a fancy rest api.